### ISSUE TITLE
about the dataset

### ISSUE THREAD

**Issue by jiejiangd** [Abenezergirma/Aircraft-Localization-using-ADS-B-data#1]
I would like to ask if the data set is publicly available

**Comment by Abenezergirma** [comment]
Yes, it is publicly available (at least at the time I was working on this project) @ https://opensky-network.org/. You probably need to create an account to access the dataset.
